setup.cfg

```diff
@@ -20,6 +20,7 @@ exclude =
     _vendor,
     data
 select = E,W,F
+ignore = W504
```

Comment:
another option is to remove the `select` and allow the default select / ignore to take -- by default `W504` (and W503) are disabled

Reply:
> (*) In the default configuration, the checks E121, E123, E126, E133, E226, E241, E242, E704, W503, W504 and W505 are ignored 

I'd prefer to just disable `W504` in this case. If it proves to be an issue, that's for a follow up. :3